build_docker.sh

```diff
@@ -133,11 +134,21 @@ fi
 ## Push
 if [ -n "${IS_PUSH}" ]; then
 
+	echo "Pushing to ${REPO_PRJ}"
 	docker push ${REPO_PRJ}:${GITHUB_TAG}
 
+	echo "Pushing to ${GCR_REPO}"
+	docker tag ${REPO_PRJ}:${GITHUB_TAG} ${GCR_REPO}:${GITHUB_TAG}
+	gcloud docker -- push ${GCR_REPO}:${GITHUB_TAG}
+
 	if [ -z "${IS_NOT_LATEST}" ] && [ -z "${IS_HASH}" ] ; then
+		echo "Updating latest tag in ${REPO_PRJ}"
 		docker build -t ${REPO_PRJ}:latest .
 		docker push ${REPO_PRJ}:latest
+		
+		echo "Updating latest tag in ${GCR_REPO}"
```

Comment:
You should consider testing this branch by changing this tag from latest and making sure it uploads on the GCR repo.